core/src/replay_stage.rs

```diff
@@ -474,6 +474,32 @@ impl ReplayStage {
                     total_epoch_stakes,
                     stake_lockout.lockout(),
                 );
+                let percentage_stake: f64 =
```

Comment:
So this represents a node's perspective on how much total stake a particular fork has? I was thinking each node would be reporting its own contribution by reporting when it voted on a particular fork along with how much stake it placed on that vote. I'm not familiar with how stake lockouts work yet though 😅

Reply:
Yes, exactly.
Getting a reliable figure by having each node report its contribution would depend on all validators reporting to this metrics server. This way, we are seeing the actual consensus data seen by any node who choose to report.

Reply:
Right, can't rely on all nodes to report that, I should know that! 😆 Thanks!